<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title>胡耀华 - 软件</title>
</head>
<body>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-category">主页</div>
<div class="menu-item"><a href="index.html">首页</a></div>
<div class="menu-item"><a href="Education.html">学历</a></div>
<div class="menu-item"><a href="../index.html">English</a></div>
<div class="menu-category">研究</div>
<div class="menu-item"><a href="Publications.html">论文</a></div>
<div class="menu-item"><a href="Grants.html">项目</a></div>
<div class="menu-item"><a href="Software.html" class="current">软件</a></div>
<div class="menu-item"><a href="Conference.html">会议</a></div>
</td>
<td id="layout-content">
<div id="toptitle">
<h1>胡耀华 - 软件</h1>
</div>
<h2>Matlab</h2>
<ul>
<li><p><a href="http://jjwanglab.org/LpRGNI/">LpRGNI (Lp Regularization for Gene Network Inference)</a><br />
- LpRGNI是运用Lasso类方法（包括<img class="eq" src="eqs/1533415666-130.png" alt="ell_0" style="vertical-align: -4px" />, <img class="eq" src="eqs/1533415667-130.png" alt="ell_1" style="vertical-align: -3px" />和<img class="eq" src="eqs/1967866126-130.png" alt="ell_{1/2}" style="vertical-align: -8px" />正则化方法），结合高通量基因表达数据和转录因组学数据来预测基因调控网络的Matlab工具包。<br />
- 参考文献:<br /></p>
<ul>
<li><p>J. Qin, Y. Hu, F. Xu, H. K. Yalamanchili and J. Wang, Inferring gene regulatory networks by integrating ChIP-seq/chip and transcriptome data via LASSO-type regularization methods, Methods, 67: 294-303, 2014. [<a href="./pdf/2014_IGRN-Lp-Methods_QHXYW.pdf">link</a>] </p>
</li>
</ul>

</li>
</ul>
<h2>R</h2>
<ul>
<li><p><a href="https://CRAN.R-project.org/package=GSparO">GSparO (Group Sparse Optimization)</a><br />
- GSparO是运用proximal gradient algorithm求解组稀疏优化的<img class="eq" src="eqs/1767278727-130.png" alt="ell_{p,q}" style="vertical-align: -6px" />正则化模型的R工具包。<br />
- 参考文献:<br /></p>
<ul>
<li><p>Y. Hu, C. Li, K. Meng, J. Qin and X. Yang, Group sparse optimization via <img class="eq" src="eqs/1767278727-130.png" alt="ell_{p,q}" style="vertical-align: -6px" /> regularization, Journal of Machine Learning Research, 18(30): 1-52, 2017. [<a href="./pdf/2017_GSO-JMLR_HLMQY.pdf">link</a>] </p>
</li>
</ul>

</li>
</ul>
<h2>Webserver</h2>
<ul>
<li><p><a href="http://qinlab.sls.cuhk.edu.hk/CrusTF/">CrusTF (Crustacean Transcription Factors)</a><br />
- CrusTF是研究甲壳类动物进化和功能的转录因子的网络服务器。<br />
- 参考文献:<br /></p>
<ul>
<li><p>J. Qin, Y. Hu, K. Ma, X. Jiang, C. Ho, L. Tsang, L. Yi and K. Chu, CrusTF: A comprehensive resource for evolutionary and functional studies of crustacean transcription factors, BMC Genomics, 18(1): 908, 2017. [<a href="./pdf/2017_CrusTF-BMCGenomics_QHC.pdf">link</a>]</p>
</li>
</ul>

</li>
</ul>
</td>
</tr>
</table>
</body>
</html>
